aea/helpers/base.py

```diff
@@ -953,7 +953,7 @@ def json(self) -> Dict:
             not_after=self._not_after_string,
             public_key=self.public_key_or_identifier,
             message_format=self.message_format,
-            save_path=str(self.save_path),
+            save_path=str(self.save_path.as_posix()),
```

Comment:
There's a wider issue here. Reviewing `aea/configurations/base.py`'s `json` properties there is multiple non-deterministic fields. I suggest we fix all fields as otherwise we're just kicking the can down the road.

Fields that look problematic: 
- https://github.com/valory-xyz/open-aea/blob/main/aea/configurations/base.py#L704
- https://github.com/valory-xyz/open-aea/blob/main/aea/configurations/base.py#L705
- https://github.com/valory-xyz/open-aea/blob/main/aea/configurations/base.py#L709
- check https://github.com/valory-xyz/open-aea/blob/main/aea/configurations/base.py#L714
- https://github.com/valory-xyz/open-aea/blob/main/aea/configurations/base.py#L1063
- https://github.com/valory-xyz/open-aea/blob/main/aea/configurations/base.py#L1068-L1070
- https://github.com/valory-xyz/open-aea/blob/main/aea/configurations/base.py#L1418
- https://github.com/valory-xyz/open-aea/blob/main/aea/configurations/base.py#L1427
- https://github.com/valory-xyz/open-aea/blob/main/aea/configurations/base.py#L1428-L1434
- https://github.com/valory-xyz/open-aea/blob/main/aea/configurations/base.py#L1715-L1718

Similar path issues:
- https://github.com/valory-xyz/open-aea/blob/main/aea/configurations/base.py#L854
- https://github.com/valory-xyz/open-aea/blob/main/aea/configurations/base.py#L856
- https://github.com/valory-xyz/open-aea/blob/main/aea/configurations/base.py#L1440
- https://github.com/valory-xyz/open-aea/blob/main/aea/configurations/base.py#L1830-L1832